### PR TITLE
add gmx_mpi

### PR DESCRIPTION
**Pull request by jamesmkrieger**
This should be helpful for running multiple simulations together and using clusters

### CODE REVIEW

**Review by DaniDelHoyo — CHANGES_REQUESTED** [review]

File: gromacs/protocols/protocol_MD_simulation.py [hunk 1]
```diff
@@ -588,7 +593,7 @@ def callGROMPP(self, mdpFile):
         print('{} warnings in stage {}'.format(nWarns, stageNum))
         if nWarns >= 1:
             command += ' -maxwarn {}'.format(nWarns)
-        gromacsPlugin.runGromacs(self, 'gmx', command, cwd=stageDir)
+        gromacsPlugin.runGromacs(self, 'gmx', command, cwd=stageDir, numberOfMpi=0)
```
[the remark this answers is not in the document]
Comment (jamesmkrieger) [reply gromacs/protocols/protocol_MD_simulation.py:596]: No, this is a parameter for runJob to not use mpi for grompp even though we have mpi set overall. The mpi option for runGromacs is already False by default

[the remark this answers is not in the document]
Comment (DaniDelHoyo) [reply gromacs/protocols/protocol_MD_simulation.py:596]: runGromacs does not have a **kwargs so I think passing that argument makes no difference

[the remark this answers is not in the document]
Comment (jamesmkrieger) [reply gromacs/protocols/protocol_MD_simulation.py:596]: oh, in that case we should add it

[the remark this answers is not in the document]
Comment (jamesmkrieger) [reply gromacs/protocols/protocol_MD_simulation.py:596]: done

File: gromacs/__init__.py [hunk 2]
```diff
@@ -63,14 +63,22 @@ def defineBinaries(cls, env):
...
 		# Instantiating install helper
 		installer = InstallHelper(GROMACS_DIC['name'], cls.getVar(GROMACS_DIC['home']), GROMACS_DIC['version'])
 
 		# Defining some variables
 		gromacsFileName = f"gromacs-{GROMACS_DIC['version']}.tar.gz"
+		
 		charmInnerLocation = join('share', 'top')
 		charmFileName = 'charmm36-feb2021.ff.tgz'
 
+		normalInnerLocation = join('build')
```
[the remark this answers is not in the document]
Comment (jamesmkrieger) [reply gromacs/__init__.py]: done

**Comment by MartinSalinas98** [comment]
Creating new Pull Request to monitor tests in Github Actions. If this PR then does not make any test fail (if they did not fail already on devel), it is good to merge

**Review by MartinSalinas98 — APPROVED** [review]

**Comment by DaniDelHoyo** [comment]
If we think everything is correct, I'll merge it soon, together with the rest of plugins where I have modified the absolute paths in the output to be relative

File: gromacs/objects/data.py [hunk 3]
```diff
@@ -86,7 +86,7 @@ def readTrjInfo(self, protocol, outDir=None):
         outDir = os.path.dirname(self.getSystemFile()) if not outDir else outDir
         infoFile = os.path.abspath(protocol._getPath('logs/run.stderr'))
 
-        command = 'check -f {}'.format(self.getTrajectoryFile())
+        command = 'check -f {}'.format(os.path.abspath(self.getTrajectoryFile()))
```
Comment (jamesmkrieger) [inline gromacs/objects/data.py:89]: this should be relpath?

**Comment by jamesmkrieger** [comment]
Seems good to me. I added one comment where you put abspath not relpath and that's it

File: gromacs/objects/data.py (hunk 3 above)
Comment (DaniDelHoyo) [reply gromacs/objects/data.py:89]: not here, the problem with the abspaths are in the output sqlites, in order to be able to migrate them to other machines, they should be relative. However, some executions (like this one) do need an abspath (or to configure the cwd and adjust the relpaths, which might be challenging in some cases, so abspath is easier)

**Review by DaniDelHoyo — APPROVED** [review]

File: gromacs/objects/data.py (hunk 3 above)
Comment (jamesmkrieger) [reply gromacs/objects/data.py:89]: ok, makes sense. Thanks

**Pull request merged**